### PR TITLE
Pa01

### PR DESCRIPTION
**Pull request by Havardei96**
Olav Espenes and Håvard Eig snakes and ladders

### CODE REVIEW

**Review by OlavEspenes — APPROVED** [review]
Good work.

File: src/pa01/snakes_and_ladders.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import numpy as np
+
+
+def single_game(num_players1):
+    fixed_position = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85,
+                      24: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12, 87: 70}
+    position = [0] * num_players1
+    while_playing = True
+    moves = 0
+    while while_playing:
```
Comment (huynhngoc) [inline src/pa01/snakes_and_ladders.py:17]: `while max(position) < 90:`

File: src/pa01/snakes_and_ladders.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    """
+    random.seed(seed1)
+
+    return multiple_games(num_games3, num_players3)
+
+
+if __name__ == '__main__':
+    seed = 12345
+    num_players = 4
+    num_games = 100
```
Comment (huynhngoc) [inline src/pa01/snakes_and_ladders.py:76]: This is the only reason you get the warning for outerscope variables, change this then you can make the names of arguments for others function normally (without adding weird number at the end)

**Review by huynhngoc — APPROVED** [review]
Nice work.
However, @OlavEspenes did you work in this pull request. I only see you commit one unrelated small change? You two should switch role regularly (ie have similar number of commits and contributions) to prove that you both really work on it. I approve this time but for the next PA I would like to see commits from both of you.

**Pull request merged**